scripts/busybox_test.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/python3
+import subprocess
+import os
+from helper_functions import *
+from test_api_calls import *
+import time
+
+
+ALCOR_SCRIPTS = os.path.dirname(os.path.abspath(__file__))
+os.chdir("..")
+ALCOR_ROOT = os.getcwd()
+ALCOR_SERVICES = ALCOR_ROOT +"/services"
+services_list = ['ignite', 'vpm', 'snm', 'rm', 'pim' ,'mm' ,'pm' ,'nm', 'sgm', 'ag', 'dpm', 'eim', 'qm', 'nam','gm']
```

Comment:
Refer https://github.com/futurewei-cloud/alcor/blob/master/docs/modules/ROOT/pages/controller.adoc for service short names.